### PR TITLE
Feature/342 job filter

### PR DESCRIPTION
**Pull request by jeanrodriguez**
Added job filter to allow the user to see more job oportunity. 

Options are : **15, 30, 50, 100**

Ticket Link : https://github.com/developersdo/empleo-dot-net/issues/342

page-size

page-size-mobile

https://github.com/developersdo/empleo-dot-net/issues/342

TEST : 

1. go to index

1. click "mostrar mas empleos"

1.  in jobs view change "cantidad de registros"

### CODE REVIEW

**Comment by Willjobs94** [comment]
Good job @jeanrodriguez, remember to include a reference to the issue you're solving on your PR description and once you're finished, do an interactive rebase to squash your commits into one with the proper naming convention.

**Comment by eatskolnikov** [comment]
@jeanrodriguez Great contribution! I liked that you respected the naming convention for the branch. I've assigned a few people to review this PR, and would like you to read this article while waiting for the approval. It talks about how to make a great PR and it is written by @claudiosanchez You may want to gain some points with him :p 
https://medium.com/@claudiosanchez/a-pull-request-that-will-make-you-smile-45d4f0f0ad18

File: EmpleoDotNet.Core/EmpleoDotNet.Core.csproj
Comment (eatskolnikov) [inline EmpleoDotNet.Core/EmpleoDotNet.Core.csproj:37]: @jeanrodriguez  ¿Qué sucede con esta referencia? ¿Para qué es?

**Review by eatskolnikov — CHANGES_REQUESTED** [review]

File: EmpleoDotNet.Core/EmpleoDotNet.Core.csproj
Comment (eatskolnikov) [reply EmpleoDotNet.Core/EmpleoDotNet.Core.csproj:37]: @jeanrodriguez este PR está super inactivo y tiene conflictos, arregla los conflictos y dime lo de esa referencia para darle merge

**Pull request closed without merging**
